README.md

```diff
@@ -371,6 +376,87 @@ Failed: text 'foo' not present on element with locator 'By(css selector, h1)'.
+| [clickWhenClickable](#clickWhenClickable)                                               | [click](#click)                                                   |
+| [fillFieldWithTextWhenVisible](#fillFieldWithTextWhenVisible)                           | [fillFieldWithText](#fillFieldWithText)                           |
+| [fillInputFieldWithFileWhenPresent](#fillInputFieldWithFileWhenPresent)                 | [uploadFileIntoInputField](#uploadFileIntoInputField)             |
+| [clearFieldWhenVisible](#clearFieldWhenVisible)                                         | [clear](#clear)                                                   |
+| [clearFieldWhenVisibleAndFillItWithText](#clearFieldWhenVisibleAndFillItWithText)       | [clearFieldAndFillItWithText](#clearFieldAndFillItWithText)       |
+| [tapWhenTappable](#tapWhenTappable)                                                     | [tap](#tap)                                                       |
+| [fillFieldWithTextWhenVisibleAndPressEnter](#fillFieldWithTextWhenVisibleAndPressEnter) | [fillFieldWithTextAndPressEnter](#fillFieldWithTextAndPressEnter) |
+| [scrollToElementWhenVisible](#scrollToElementWhenVisible)                               | [scrollToElement](#scrollToElement)                               |
+
+#### What to do?
+
+- Only change the function on **Old** column to the matching function of **New** column, removing the `errorMessage` argument.
+  - All new methods, unlike the old ones, don't use the `errorMessage` argument.
+
+#### How I know that my code is ok?
+
+You will know that you have solved all your problems when don't see any warning on console like this:
+
+```
+Protractor-helper warning: Function 'fillFieldWithTextWhenVisible' will be deprecated in version 4.0.0! Please use the new 'fillFieldWithText' function instead! [Read more on www.npmjs.com/package/protractor-helper#preparation-to-next-major-version]
+```
+
+### Argument `errorMessage` will be removed
+
+We will remove all `errorMessage` argument from the library because we trust on our messages. They are clear and tell you exactly why the test has failed.
+
+Methods afected:
+
+1. [waitForElementPresence](#waitForElementPresence)
+2. [waitForElementNotToBePresent](#waitForElementNotToBePresent)
+3. [waitForElementVisibility](#waitForElementVisibility)
+4. [waitForElementNotToBeVisible](#waitForElementNotToBeVisible)
+5. [waitForTextToBePresentInElement](#waitForTextToBePresentInElement)
+6. [waitForTextNotToBePresentInElement](#waitForTextNotToBePresentInElement)
+7. [waitForUrlToBeEqualToExpectedUrl](#waitForUrlToBeEqualToExpectedUrl)
+8. [waitForUrlNotToBeEqualToExpectedUrl](#waitForUrlNotToBeEqualToExpectedUrl)
+9. [waitForUrlToContainString](#waitForUrlToContainString)
+10. [waitForUrlNotToContainString](#waitForUrlNotToContainString)
+
+#### What to do?
+
+- Remove all `errorMessage` argument from all functions above to don't have trouble on next major version (v4.0.0).
```

Comment:
What about: `Remove all `errorMessage` arguments to avoid breakages in the next major version (v4.x.x).` ?